packages/docusaurus-theme-common/package.json

```diff
@@ -36,6 +36,7 @@
     "@docusaurus/plugin-content-docs": "^3.0.0-alpha.0",
     "@docusaurus/plugin-content-pages": "^3.0.0-alpha.0",
     "@docusaurus/utils": "^3.0.0-alpha.0",
+    "@docusaurus/theme-classic": "^3.0.0-alpha.0",
```

Comment:
Looks suspicious, theme-common should not import theme-classic or it would create a cyclic dependency and our build can't know which package to build first